sdk/nodejs/entrypoint/load.ts

```diff
@@ -19,17 +19,31 @@ export async function load(files: string[]): Promise<void> {
  */
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 export async function loadArg(value: string): Promise<any> {
-  const trimmedValue = value.slice(1, value.length - 1)
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  let parsedValue: any
 
-  const [source] = trimmedValue.split(":")
-
-  const [origin, type] = source.split(".")
-  if (origin === "core") {
-    // Workaround to call get any object that has an id
-    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-    // @ts-ignore
-    return dag[`load${type}FromID`](trimmedValue as ID)
+  // Apply JSON parse to parse array or string if the value is wrapped into a string or array
+  if (
+    (value.startsWith('"') && value.endsWith('"')) ||
+    (value.startsWith("[") && value.endsWith("]"))
```

Comment:
I hope you refactor `loadArg` based on expected type rather than value content, but I won't block it for now.

Reply:
You just need to have the introspection results by the time you invoke. We'll work on that.